packages/app/src/device-manager/types/port.type.ts

```diff
@@ -5,4 +5,5 @@
 
 import { PortInfo as SerialPortInfo } from "serialport"
 
-export type PortInfo = Omit<SerialPortInfo, "path">
+// export type PortInfo = Omit<SerialPortInfo, "path">
```

Comment:
to be removed, i guess 🤔

Reply:
fixed 9e2d742a8b4d419f433d38b40c08828f004c23bf